src/Microsoft.Health.Dicom.Core/Features/Operations/IDicomOperationsClient.cs

```diff
@@ -116,7 +117,7 @@ public interface IDicomOperationsClient
     /// <paramref name="updateSpecification"/> is <see langword="null"/>.
     /// </exception>
     /// <exception cref="OperationCanceledException">The <paramref name="cancellationToken"/> was canceled.</exception>
-    Task<OperationReference> StartUpdateOperationAsync(Guid operationId, UpdateSpecification updateSpecification, int partitionKey, CancellationToken cancellationToken = default);
+    Task<OperationReference> StartUpdateOperationAsync(Guid operationId, UpdateSpecification updateSpecification, int partitionKey, string endpoint, CancellationToken cancellationToken = default);
```

Comment:
Why is this part of the function input? Can this not be part of the function app's config?

Reply:
We have the endpoint available in web, but not in functions. So the endpoint is grabbed from the requestContext when the user queues an update operation

Reply:
Oh, so we need to log the entire URL? Not just the host?